prow/plugins/cat/cat.go

```diff
@@ -201,5 +201,10 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, c
 		return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
 	}
 
+	if category != "" {
```

Comment:
cant we also hit this if we just fail to reach the cat api 3 times?
could hoist err out of the loop and use `if err == nil && category != ""`

/shrug

Reply:
err isn't defined in this scope

Reply:
yes, but we could increase the scope for err above
/shrug

Reply:
Done